tests/pdctl/member/member_test.go

```diff
@@ -117,3 +119,76 @@ func (s *memberTestSuite) TestMember(c *C) {
 	c.Assert(len(members.Members), Equals, 2)
 	c.Succeed()
 }
+
+func (s *memberTestSuite) TestUpdateMemberDCLocationInfo(c *C) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	dcLocationConfig := map[string]string{
+		"pd1": "dc-1",
+		"pd2": "dc-2",
+		"pd3": "dc-3",
+	}
+	cluster, err := tests.NewTestCluster(ctx, 3, func(conf *config.Config, serverName string) {
+		conf.LocalTSO = config.LocalTSOConfig{
+			EnableLocalTSO: true,
+			DCLocation:     dcLocationConfig[serverName],
+		}
+	})
+	c.Assert(err, IsNil)
+	err = cluster.RunInitialServers()
+	c.Assert(err, IsNil)
+	cluster.WaitLeader()
+	defer cluster.Destroy()
+
+	pdAddr := cluster.GetConfig().GetClientURL()
+	cmd := pdctl.InitCommand()
+
+	checkDCLocation := func(expected, unexpected []string) {
+		testutil.WaitUntil(c, func(c *C) bool {
+			args := []string{"-u", pdAddr, "member"}
+			_, output, err := pdctl.ExecuteCommandC(cmd, args...)
+			c.Assert(err, IsNil)
+			resp := &pdpb.GetMembersResponse{}
+			c.Assert(json.Unmarshal(output, resp), IsNil)
+			c.Log(fmt.Sprintf("allocator leaders: %s", resp.GetTsoAllocatorLeaders()))
+
+			for _, l := range unexpected {
+				_, ok := resp.GetTsoAllocatorLeaders()[l]
+				if ok {
+					return false
+				}
+			}
+			for _, l := range expected {
+				_, ok := resp.GetTsoAllocatorLeaders()[l]
+				if !ok {
+					return false
+				}
+			}
+			return true
+		}, testutil.WithSleepInterval(time.Second))
+	}
+
+	// check initial dc location
+	checkDCLocation([]string{"dc-1", "dc-2", "dc-3"}, []string{})
+
+	// modify dc location of pd1
+	args := []string{"-u", pdAddr, "member", "dc_location", "name", "pd1", "dc-4"}
```

Comment:
I prefer `dc_location` as an independent sub command like:
```
pd-ctl -u <pd-addr> location <name> <dc>
```
which is similar like:

```sh
pd-ctl -u <pd-addr> label <store> <key> <value>
```